src/components/category-menu/index.css

```diff
@@ -38,6 +37,11 @@
 .category-menu-mobile {
     overflow-x: scroll;
     overflow-y: hidden;
+    height: 70.16px !important;
```

Comment:
Not required. The current height is okay in my opinion.

Reply:
This line was an attempt to hide the horizontal scroll bar, which did not look good. The earlier attempt was using a blank icon that had the same height but caused that unnecessary left margin which the bug was all about. Removing the line will bring the scroll bar in the view that's all.

Reply:
You can use `overflow-x: hidden`

Reply:
No, we don't have to deny the scrolling. How will the user access other categories? What I meant is that this line was an attempt to keep its height to its former self. Earlier height was defined by the height of the blank icon only. Wrapper-mobile has a height of 60 px but the div with .category-menu-mobile class' height was defined by a blank icon which caused an overflow in y and hence hid the scroll bar. All I did is that I calculated the original height of  .category-menu-mobile class' div and added that in the css and hence removing the need of any blank icon. If I remove this line not only the height of navbar will become less than original but also a scroll bar will be visible.

Reply:
@gagansh7171 
Can you see if this CSS works as expected.
Keeping the height fixed here is not an excellent choice
.category-menu-mobile {
    overflow-x: scroll;
    overflow-y: hidden;
    scrollbar-width: none;
}

.category-menu-mobile::-webkit-scrollbar {
    background: transparent; 
}
Other changes are all the best they can be

Reply:
That's getting overridden and !important is not helping as well.

Reply:
> That's getting overridden and !important is not helping as well

How was it looking 
> Can you help me know why fixed height is not a good choice in this case
Our target is to remove the horizontal scrollbar and allow scrolling. So, just for hiding the scrollbars, we should not achieve it by changing height but by directly reducing their width to zero and making them transparent or something similar.

Reply:
@SuyashSalampuria  It was looking like the first photo in [this](https://github.com/IMGIITRoorkee/omniport-frontend-buy-and-sell/pull/10#discussion_r495419153) comment

Reply:
I have used webkit-scrollbar css property of `display:none` to hide the scrollbar instead of hard-coding height.